.gitignore

```diff
@@ -0,0 +1,32 @@
+# Generated code
```

Comment:
Add this set of files https://github.com/googleapis/java-tasks/blob/master/.gitignore#L21 (line 21-28) to here

Reply:
Done.